fracture_sets.py

```diff
@@ -434,7 +434,7 @@ def _define_centers_by_boxes(self, domain, distribution="poisson", intensity=Non
             return ValueError("Only Poisson point processes have been implemented")
         # Intensity scaled to this domain
         if intensity is None:
-            intensity = self.intensity * self.domain_measure(domain)
+            intensity = self.intensity #* self.domain_measure(domain)
```

Comment:
I think it is better to include the area in the variable "area_of_box" ( see comment below). But maybe I misunderstand what is meant by intensity